canvas.gypi

```diff
@@ -0,0 +1,40 @@
+{
+    'variables':
+    {
+        'has_FBDev%': 'true',
+        'has_X11%': '<!(./util/has_lib.sh X11)'
```

Comment:
How about detecting if the OS is linux and then enable fbdev and only then check if x11 is actually present.
Otherwise it would cause a build failure on any non linux devices or wouldn't it?